django/db/models/constraints.py

```diff
@@ -66,14 +72,22 @@ def deconstruct(self):
         return path, args, kwargs
 
 
+NOT_DEFERRABLE = ''
+DEFERRABLE_INITIALLY_DEFERRED = 'DEFERRABLE INITIALLY DEFERRED'
+DEFERRABLE_INITIALLY_IMMEDIATE = 'DEFERRABLE INITIALLY IMMEDIATE'
```

Comment:
I'd avoid allowing passing any form of raw SQL here, I'd suggest the constants are some form of slugs and that the actual SQL generation is handled at the backend level.